ngraph/test/frontend/frontend_manager.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
2021 and so on

Reply:
no, the dates are valid for the whole product, but not a single file. So, 2018-2021 should be everywhere.

Reply:
see https://github.com/openvinotoolkit/openvino/pull/4974/files